apis/r/README.md

```diff
@@ -7,12 +7,12 @@ Please also see the `main-old` branch for an implementation of the [original spe
 
 # Installation
 
-## Using R-universe
+## Release packages
 
-TileDB-SOMA is available on  R-universe and [Conda](https://anaconda.org/tiledb/r-tiledbsoma), and can be installed directly from R or `mambda` as indicated below.
+TileDB-SOMA releases are available on R-universe and [Conda](https://anaconda.org/tiledb/r-tiledbsoma), and can be installed directly from R or `mamba` as indicated below.
 
 ```r
-install.packages('tiledbsoma', repos = c('https://tiledb-inc.r-universe.dev',
+install.packages('tiledbsoma', repos = c('https://chanzuckerberg.r-universe.dev',
                                          'https://cloud.r-project.org'))
 ```
```

Comment:
It may make sense to mention that r-universe has macOS binaries (as well as Ubuntu jammy for R 4.3.1).  It would have windows binaries if we built for Windows.  

Conda/mamba exists to and for multiple architectures but users who are "mostly R users"  (and are not also Python users in corporate settings) generally do not do conda.